splice_spark/pom.xml

```diff
@@ -224,8 +224,32 @@
                 <version>1.5.0.Final</version>
             </extension>
         </extensions>
-        <plugins>
-            <plugin>
+	<plugins>
+		<plugin>
+			<groupId>org.apache.maven.plugins</groupId>
+			<artifactId>maven-site-plugin</artifactId>
+			<version>3.0</version>
```

Comment:
I think this and the other plugin versions should all be in one single place under `pluginManagement` in the top-level pom.xml. That way we cut the time needed to download all deps for the build (not to mention that there'll be no dups).